src/Core/Services/TypeHelper.cs

```diff
@@ -304,12 +305,12 @@ public static bool TryGetDbTypeFromSqlDbDateTimeType(SqlDbType sqlDbType, [NotNu
             SyntaxKind valueKind = node.Kind;
             return valueKind switch
             {
-                SyntaxKind.IntValue => Convert.ToInt32(node.Value), // spec
-                SyntaxKind.FloatValue => Convert.ToDouble(node.Value), // spec
-                SyntaxKind.BooleanValue => Convert.ToBoolean(node.Value), // spec
-                SyntaxKind.StringValue => Convert.ToString(node.Value), // spec
+                SyntaxKind.IntValue => Convert.ToInt32(node.Value, CultureInfo.InvariantCulture), // spec
+                SyntaxKind.FloatValue => Convert.ToDouble(node.Value, CultureInfo.InvariantCulture), // spec
+                SyntaxKind.BooleanValue => Convert.ToBoolean(node.Value, CultureInfo.InvariantCulture), // spec
```

Comment:
why do you need to define `CultureInfo.InvariantCulture` for Convert.ToBoolean? that parameter is ignored per documentation:https://learn.microsoft.com/dotnet/api/system.convert.toboolean?view=net-8.0#system-convert-toboolean(system-string-system-iformatprovider)